packages/uniorg/package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "uniorg",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

we must not update version fields manually because lerna would do that during release procedure
```suggestion
  "version": "0.5.0",
```

---

Whoops! Totally forgot that I did that, good that you actually have that automated hahaha